package.json

```
@@ -6,14 +6,15 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "eslint app components hooks lib -c eslint.config.mjs && prettier --check \"app/**/*.ts\" \"app/**/*.tsx\" \"components/**/*.tsx\" \"hooks/**/*.tsx\" \"lib/**/*.ts\""
```

すみません、こちらもう少しシンプルなバージョンを別のプロジェクトから引っ張ってきたので、こちらでお願いします(* ᴗ ᴗ)⁾⁾
お手数おかけして申し訳ないです。。あとあると便利なので fix コマンドも書いておきますね
```suggestion
    "lint": "eslint src test -c eslint.config.mjs && prettier --check \"src/**/*.ts\" \"test/**/*.ts\"",
    "fix": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" && eslint --fix src/**/*.ts test/**/*.ts -c eslint.config.mjs",
```

---

@Harineko0 

ありがとうございます
修正しました